lib/substitutions.json

```diff
@@ -2485,6 +2522,11 @@
     {
       "before": "decisions:  Holding back",
       "after": "decisions: holding back"
+    },
+    {
+      "before": "their positioning, they had planned this",
+      "after": "their positioning… they had planned this",
+      "_comment": "A comma seems like the wrong choice here."
```

Comment:
I agree with your assessment, but in this case we need to capitalize "They" since it's the start of a separate sentence.